decisions/components-preview.md

```diff
@@ -0,0 +1,14 @@
+# Component Preview (a.k.a stories)
+
+## Decision
+- All previews should respond to a shortcut to hide the navbar (CMD+D).
+- The component being previewed should fit in the screen in portrait mode *without* the navbar.
+- The IOS preview should be as close as possible as the Android version (Always respecting any platform specific behaviour).
+- Diferent configurations (e.g. normal text and emphasized text) should have different previews. In this case clicking the main story should show a new list view with all configs.
```

Comment:
* `Different` is misspelled.
* Nitpick: `emphasized` should be `emphasised` (British spelling).